packages/wallet-backend-update/src/rapyd/controller.ts

```diff
@@ -10,7 +10,7 @@ interface IRapydController {
   getDocumentTypes: ControllerFunction<RapydDocumentType[]>
   createWallet: ControllerFunction<RapydWallet>
   verifyIdentity: ControllerFunction<RapydIdentityResponse>
-  updateProfile: ControllerFunction<RapydWallet>
+  updateProfile: ControllerFunction
```

Comment:
we don't need a RapydWallet types response anymore?

Reply:
It was returning the whole user object from Rapyd, which we do not really need.